### PR TITLE
chore: only log SQL statements when at trace

### LINKED ISSUE #181: fix: logging broken
Actual behavior: some logs are multiline
Expected behavior: in one line

Screenshot 2023-08-18 at 2 50 49 PM

### PR DESCRIPTION
**Pull request by HarryET**
# Description
Makes sure we dont have weird error/debug logs

Resolves #181 

## How Has This Been Tested?
N/a changes log levels

## Due Diligence

* [ ] Breaking change
* [ ] Requires a documentation update
* [ ] Requires a e2e/integration test update

### CODE REVIEW

File: src/error.rs
Comment (xav) [inline src/error.rs:1]: If we won't use this, let's remove it instead of commenting it out. Otherwise, add a TODO comment please.

**Review by xav — APPROVED** [review]
See the Clippy comments too

File: src/error.rs
Comment (HarryET) [reply src/error.rs:1]: This was from testing and shouldn't have been included

**Comment by HarryET** [comment]
Local git was commiting to main so this was pushed into main, fixed main as well as my local git

**Pull request merged**